Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -961,6 +1068,20 @@ public void SwitchCurrentControlScheme(string controlScheme, params InputDevice[
             SwitchControlSchemeInternal(ref scheme, devices);
         }
 
+        /// <summary>
+        /// Switch the player to use the given action map
+        /// </summary>
+        /// <param name="mapNameOrId">Name of the action map or its ID.</param>
+        /// <remarks>
+        /// This method can be used to explicitly set an action map
+        ///
+        /// <example>
+        /// <code>
+        /// PlayerInput.all[0].SwitchCurrentActionMap("Player");
```

Comment:
Similar comment as previously on non-compilable example

Reply:
It should compile but not in isolation. I'll await doc writer feedback